src/supernode/src/test/java/com/alibaba/dragonfly/supernode/common/util/RangeParseUtilTest.java

```diff
@@ -0,0 +1,61 @@
+package com.alibaba.dragonfly.supernode.common.util;
+
+import com.alibaba.dragonfly.supernode.common.util.RangeParseUtil;
+import com.diffblue.deeptestutils.Reflector;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class RangeParseUtilTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
```

Comment:
I like these testcases! Do we cover the exception cases to cover the whole method?

Reply:
Both cases Exceptional 👍

Reply:
👍